int32 yaw_mode
int32 roll_mode
int32 ra_1
int32 ra_2
int32 ra_3
int32 ra_4
int32 ra_5
int32 ra_6